scripts/boot_patch.sh

```diff
@@ -6,7 +6,8 @@
 # Usage: boot_patch.sh <bootimage>
 #
 # The following flags can be set in environment variables:
-# KEEPVERITY, KEEPFORCEENCRYPT, PATCHVBMETAFLAG, RECOVERYMODE, SYSTEM_ROOT
+# KEEPVERITY, KEEPFORCEENCRYPT, PATCHVBMETAFLAG, RECOVERYMODE, SYSTEM_ROOT,
+# DYNAMIC_PARTITIONS
```

Comment:
There are other lines also long, might want to make this comment in a single line for easier grepping dunno.

```suggestion
# KEEPVERITY, KEEPFORCEENCRYPT, PATCHVBMETAFLAG, RECOVERYMODE, SYSTEM_ROOT, DYNAMIC_PARTITIONS
```

Reply:
It's customary to try keeping lines below 80 characters, I don't think this is an issue